DemoqaProject/tests/FormsTest/PracticeFormTest.cs

```diff
@@ -0,0 +1,43 @@
+using DemoqaProject.PageObjects;
+using NUnit.Framework;
+
+namespace DemoqaProject
+{
+    public class PracticeFormTest : BaseSeleniumTest
+    {
+        private string firstname = "Andrew";
+        private string lastname = "Garfield";
+        private string useremail = "Andrew@gmail.com";
+        private string usernumber = "9987773333";
+        private string subject = "Computer";
+        private string address = "Chicago, USA";
+        private string state = "ncr";
+        private string city = "Delhi";
+        
+        private string usernumber2 = "";
+        
+        [Test]
+        public void ValidCredentialTest()
+        {
+            HomePage homePage = new HomePage();
+            homePage.NavigateToFormsPage();
+            PracticeForm practiceForm = new PracticeForm();
+            practiceForm.NavigateToPracticeForm();
+            practiceForm.FillRegistrationForm(firstname, lastname, useremail, usernumber, subject, address, state, city);
+            
+            Assert.IsTrue(practiceForm.CheckClassExists());
+        }
+        
+        [Test]
+        public void InValidCredentialTest()
+        {
+            HomePage homePage = new HomePage();
+            homePage.NavigateToFormsPage();
+            PracticeForm practiceForm = new PracticeForm();
+            practiceForm.NavigateToPracticeForm();
+            practiceForm.FillRegistrationForm(firstname, lastname, useremail, usernumber2, subject, address, state, city);
+            
+            Assert.IsFalse(practiceForm.CheckClassExists());
+        }
```

Comment:
Please make one parametrized testcase as steps are common for both of them.
Assertion check could be done also from testcase parameter.